antarest/study/service.py

```diff
@@ -560,44 +567,72 @@ def sync_studies_on_disk(self, folders: List[StudyFolder]) -> None:
                     and study.path not in paths
                 )
             ):
-                logger.info(
-                    "Study=%s is not present in disk and will be deleted",
-                    study.id,
-                )
-                self.event_bus.push(
-                    Event(
-                        type=EventType.STUDY_DELETED,
-                        payload=study.to_json_summary(),
-                        permissions=create_permission_from_study(study),
+                if not study.missing:
```

Comment:
This looks like a boolean. missing_since should be more appropriated
and change the condition to "if study.missing_since is not None" to be even more clear